features/support/env.rb

```diff
@@ -0,0 +1,26 @@
+require 'os'
+
+Before do
+  find_default_docker_compose
+end
+
+def install_current_branch(fixture)
```

Comment:
This is brittle as it wont test local changes. Why not use the path options to composer -> https://getcomposer.org/doc/05-repositories.md#path

Reply:
Composer won't install a parent directory as a package, which it would be in this case.

Reply:
why dont you cp into another directory so its not parent?